1_Entregable_proyecto/1_1_4_Requerimientos_no_funcionales.md

```diff
@@ -0,0 +1,302 @@
+de aceptación -->
+
+* Eficiencia de uso: Qué tan rápido o con qué precisión el usuario puede usar el
+  producto.
+* Facilidad de memorización: Qué tanto se espera que el usuario casual memorice
+  en cuanto a usar el producto.
+* Tasa de errores: Qué tantos errores sería aceptable que el usuario cometa.
+  Para algunos productos, es crucial que el usuario cometa muy pocos errores, o
+  en ocasiones, ninguno.
+* Satisfacción general al usar el producto: Para productos que tienen cierta
+  competencia, es esencial que los usuarios se sientan muy satisfechos al usar
+  el producto. De ser el caso, es valioso registrar esto como un requerimiento.
+* Retroalimentación: Qué tanta retroalimentación el usuario necesita para
+  sentirse confiado en cuanto a que el producto está haciendo precisa y
+  correctamente lo que se espera de él. Productos de seguridad o tareas críticas
+  suelen necesitar un alto nivel de retroalimentación.
+
+<!-- SECCIÓN: Requerimientos de aprendizaje -->
+Siguiendo el punto anterior, también detallar las expectativas del cliente en
+cuanto a qué tanto esfuerzo deberán emplear los usuarios finales para poder
+utilizar el sistema de una forma productiva. Algunos sistemas deben ser muy
+fáciles de utilizar y por ende intuitivos, otros se pueden permitir determinada
+curva de aprendizaje debido a la complejidad del problema. La idea aquí es
+cuantificar la cantidad de tiempo o esfuerzo que los usuarios finales necesitan
+para aprender a usar el sistema y que esta cuantificación sea aceptable para el
+cliente. Esta clarificación puede incluir un plan de capacitación para los
+usuarios, e incluso un manual de usuario, por ejemplo: "Los usuarios deberán ser
+capaces de usar el sistema tras una semana de capacitación" o, "Los usuarios
+deberán ser capaces de utilizar el sistema tras leer el manual de usuario".
+
+<!-- SECCIÓN: Requerimientos de personalización e internacionalización -->
+En caso de que sea establecido por el cliente como una necesidad, especificar
+los requerimientos asociados a las preferencias de los usuarios. Estos
+requerimientos indican cómo el sistema puede ser modificado o configurado por
+los usuarios para tener en cuenta sus preferencias y selección de lenguaje
+utilizado por el sistema. En otras culturas, se usan diferentes sistemas de
+unidades —sistema métrico, anglosajón, etcétera—, unidades monetarias, idiomas,
+etcétera.
+
+<!-- SECCIÓN: Requerimientos de accesibilidad -->
+Especificar los requerimientos asociados a la accesibilidad del sistema.
+Dependiendo del problema y el producto, este tipo de requerimientos puede ser
+más necesario.
+
+<!-- SECCIÓN: Requerimientos de protección crítica -->
+En caso de que usar el producto desarrollado signifique un posible riesgo de
+daño percibido a personas o propiedad dentro del entorno de uso, detallar los
+requerimientos que cuantifican este daño. Estos pueden estar relacionados a
+estándares de seguridad o protección.
+
+## Requerimientos de *performance*
```

Comment:
```suggestion
## Requerimientos de performance
```

Reply:
No es necesario poner performance en itálica: https://dle.rae.es/performance 😉

Reply:
Capaz que podemos agregar a la sección de Conceptos las definiciones de los "-ilities" más comunes, así como formas de medirlos y de probarlos, ¿cómo lo ves?